soc/nxp/imxrt/rt6xx/Kconfig.soc

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMXRT_6XX
+	bool
+	select SOC_FAMILY_IMXRT
+
+config SOC_SERIES
+	default "rt6xx" if SOC_SERIES_IMXRT_6XX
+
+config SOC_MIMXRT685S_CM33
+	bool
+	select SOC_SERIES_IMXRT_6XX
+
+config SOC
+	default "mimxrt685s_cm33" if SOC_MIMXRT685S_CM33
```

Comment:
soc names to not have a core name in them

Reply:
this will require hal changes then, because we use CONFIG_SOC as the variable needed to build stuff in the hal, which uses this string

Reply:
nevermind we changed this